package.json

```diff
@@ -105,6 +105,8 @@
 		"@exodus/schemasafe": "^1.3.0",
 		"@sinclair/typebox": ">=0.32.30 <1",
 		"@sveltejs/kit": "1.x || 2.x",
+		"@typeschema/class-validator": "^0.1.2",
+		"class-validator": "^0.13.1",
```

Comment:
I didn't think this should be needed as there should be a transitive package reference but tests failed without it.